examples/todomvc/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "redux-todomvc-example",
+  "name": "redux-todos-example",
```

Comment:
Keep it todomvc? The `todos` example is the simpler one, with less UI fanciness.